### PR TITLE
Relies on libkiwix 14.0.0

### LINKED ISSUE #984: Add support of libkiwix14
Libkiwix 14 [is almost over](https://github.com/kiwix/libkiwix/milestone/12), we should use it when released.

Blocked by https://github.com/kiwix/libkiwix/issues/1081

### PR DESCRIPTION
**Pull request by kelson42**
Fixes #984

### CODE REVIEW

**Review by rgaudin — DISMISSED** [review]
LGTM (not tested)

**Comment by kelson42** [comment]
@BPerlakiH I guess this is your turn now to complete this PR :(

**Review by balazs-vimn — APPROVED** [review]

**Comment by kelson42** [comment]
@BPerlakiH Hmmm.... the CI does not pass... we can not merge like this

**Comment by rgaudin** [comment]
Please use https://download.kiwix.org/release/libkiwix/libkiwix_xcframework-14.0.0-1.tar.gz

File: Brewfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ brew "curl"
 
 at_exit do
     system "pre-commit install"
-    system "curl -L -o - https://download.kiwix.org/release/libkiwix/libkiwix_xcframework-13.1.0-4.tar.gz | tar -x --strip-components 2"
+    system "curl -L -o - https://download.kiwix.org/release/libkiwix/libkiwix_xcframework-14.0.0.tar.gz | tar -x --strip-components 2"
```
Comment (rgaudin) [inline Brewfile:7]: Not the right one

**Comment by kelson42** [comment]
@rgaudin @BPerlakiH CI is green! Does that mean we can merge?

**Comment by BPerlakiH** [comment]
I am testing it right now...

**Review by BPerlakiH — APPROVED** [review]

**Pull request merged**